about.html

```diff
@@ -0,0 +1,225 @@
+            />
+          </div>
+        </div>
+      </section>
+      <section class="past-summits">
+        <div class="container">
+          <h2 class="past-summits__headline small-underline">
+            See the Past African Education Summit
+          </h2>
+          <p class="past-summits__paragraph">
+            Take a look at the previous two African Education Summits held in
+            Accra and Kinshasa.
+          </p>
+          <ul class="past-summits__cards-wrapper">
+            <li class="past-summits__card">
+              <img
+                class="past-summits__card__img"
+                src="assets/images/accra-2021.jpg"
+                alt="African Education Summit in Accra 2021"
+              />
+              <div class="past-summits__card__body">
+                <h4 class="past-summits__card__header">2021</h4>
+                <p class="past-summits__card__description">
+                  African Education Summit 2021 in Accra
+                </p>
+                <button class="past-summits__card__btn" type="button">
+                  See More
+                </button>
+              </div>
+            </li>
+            <li class="past-summits__card">
+              <img
+                class="past-summits__card__img"
+                src="assets/images/kinshasa-2022.jpg"
+                alt="African Education Summit in Kinshasa 2022"
+              />
+              <div class="past-summits__card__body">
+                <h4 class="past-summits__card__header">2022</h4>
+                <p class="past-summits__card__description">
+                  African Education Summit 2022 in Kinshasa
+                </p>
+                <button class="past-summits__card__btn" type="button">
+                  See More
+                </button>
+              </div>
+            </li>
+          </ul>
+        </div>
+      </section>
+      <div id="overlay" class="overlay"></div>
+    </main>
+    <footer class="footer">
+      <div class="container">
+        <div class="footer__wrapper">
+          <a href="/" class="footer__logo__wrapper">
+            <img class="footer__logo" src="assets/icons/logo-lg.svg" alt="" />
+          </a>
+          <div>
+            <p class="footer__copyright">
+              &#169; 2023 African Education Summit. Some rights reserved.
+            </p>
+            <p class="footer__disclaimer">
+              All eniities represented on this website, including the partners,
+              speakers and program, are all fictional and were created for the
+              purpose of a web development project. The source code for this
+              website can be found
+              <a
+                class="footer__github-link"
+                href="https://github.com/henokkhm/microverse-module-1-capstone-project"
+              >
+                here </a
+              >.
+            </p>
+          </div>
+        </div>
+      </div>
+    </footer>
```

Comment:
- [ ] In the mobile version there should be a Partner section, kindly make sure to add the Partner section in the mobile version of your website so it looks identical to the provided design
- [ ] The footer logo and text should be aligned horizontally in both mobile and desktop view, kindly make sure that the footer alignment is the same on mobile and desktop so your website is the same as the provided design

|Your website|Provided design|
|-|-|
|![Screenshot_20230529_232038](https://github.com/henokkhm/microverse-module-1-capstone-project/assets/25227022/103dc60c-df5f-4cb3-9c8d-a4a814f99e72)|![Screenshot_20230529_232018](https://github.com/henokkhm/microverse-module-1-capstone-project/assets/25227022/12368f8b-068e-4815-bcbc-d65ed0680c21)|

Reply:
I think a column layout looks better on mobile
![image](https://github.com/henokkhm/microverse-module-1-capstone-project/assets/110997325/8c141421-b9d5-4e6b-8cb3-7eb711d9840c)
![image](https://github.com/henokkhm/microverse-module-1-capstone-project/assets/110997325/45f2ce4e-56d0-4905-97ea-a911b35ff5cc)
But I will fix it for the next review

Reply:
@henokkhm, you can reduce the logo size or remove some part of the paragraph, but the footer should be aligned horizontally